pkg/cloud/google/machineactuator.go

```diff
@@ -689,6 +692,10 @@ func (gce *GCEClient) updateMasterInplace(cluster *clusterv1.Cluster, oldMachine
 	return nil
 }
 
+func stripVersion(version string) string {
+	return strings.Trim(version, "vV")
```

Comment:
Probably should be `TrimLeft`, but in practice it's not going to end with v anyway